Warning messages when compiling with maven

> [!IMPORTANT]
> Read issue #2 first

![image](https://github.com/user-attachments/assets/ccb61348-75ae-4fed-aeb0-51170c688b8e)

Investigate why these messages appear.

Hi, @nmarulo, I am not able to start the application using docker and also on the local machine, 
getting some errors like, 


at org.springframework.beans.factory.support.AbstractBeanFactory.getBean(AbstractBeanFactory.java:205) ~[spring-beans-6.1.11.jar:6.1.11]
despensa-rest-api-1   | 	at org.springframework.context.support.AbstractApplicationContext.finishBeanFactoryInitialization(AbstractApplicationContext.java:954) ~[spring-context-6.1.11.jar:6.1.11]
despensa-rest-api-1   | 	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:625) ~[spring-context-6.1.11.jar:6.1.11]
despensa-rest-api-1   | 	at org.springframework.boot.web.servlet.context.ServletWebServerApplicationContext.refresh(ServletWebServerApplicationContext.java:146) ~[spring-boot-3.3.2.jar:3.3.2]
despensa-rest-api-1   | 	at org.springframework.boot.SpringApplication.refresh(SpringApplication.java:754) ~[spring-boot-3.3.2.jar:3.3.2]
despensa-rest-api-1   | 	at org.springframework.boot.SpringApplication.refreshContext(SpringApplication.java:456) ~[spring-boot-3.3.2.jar:3.3.2]
despensa-rest-api-1   | 	at org.springframework.boot.SpringApplication.run(SpringApplication.java:335) ~[spring-boot-3.3.2.jar:3.3.2]
despensa-rest-api-1   | 	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1363) ~[spring-boot-3.3.2.jar:3.3.2]
despensa-rest-api-1   | 	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1352) ~[spring-boot-3.3.2.jar:3.3.2]
despensa-rest-api-1   | 	at dev.nmarulo.depensaapp.DespensaRestApiApplication.main(DespensaRestApiApplication.java:10) ~[classes/:0.0.1]
despensa-rest-api-1   | 	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103) ~[na:na]
despensa-rest-api-1   | 	at java.base/java.lang.reflect.Method.invoke(Method.java:580) ~[na:na]
despensa-rest-api-1   | 	at org.springframework.boot.loader.launch.Launcher.launch(Launcher.java:91) ~[build/:na]
despensa-rest-api-1   | 	at org.springframework.boot.loader.launch.Launcher.launch(Launcher.java:53) ~[build/:na]
despensa-rest-api-1   | 	at org.springframework.boot.loader.launch.WarLauncher.main(WarLauncher.java:57) ~[build/:na]
despensa-rest-api-1   | Caused by: jakarta.persistence.PersistenceException: [PersistenceUnit: default] Unable to build Hibernate SessionFactory; nested exception is org.hibernate.tool.schema.spi.SchemaManagementException: Schema-validation: wrong column type encountered in column [id] in table [products]; found [int (Types#INTEGER)], but expecting [bigint (Types#BIGINT)]


Is it expected, or I have done something wrong, likely stuck on this from last night and it's afternoon now but didn't find a way to resolve this.

--- a/pom.xml
+++ b/pom.xml
@@ -92,6 +92,14 @@
             <artifactId>springdoc-openapi-starter-webmvc-ui</artifactId>
             <version>${springdoc-openapi-starter-webmvc-ui.version}</version>
         </dependency>
+
+        <dependency>
+            <groupId>com.google.code.findbugs</groupId>
+            <artifactId>jsr305</artifactId>
+            <version>3.0.2</version>
+            <scope>provided</scope>
+        </dependency>
+
     </dependencies>
 
     <build>

